modules/cms/classes/MediaLibraryItem.php

```diff
@@ -13,7 +13,7 @@ class MediaLibraryItem extends SystemMediaLibraryItem
 {
     public function __construct()
     {
-        traceLog('Class Cms\Classes\MediaLibraryItem has been deprecated, use System\Classes\MediaLibraryItem instead.');
+        traceLog('Class Cms\Classes\MediaLibraryItem has been deprecated, use ' . SystemMediaLibraryItem::class . ' instead.');
```

Comment:
Are you going to make changes to the other traceLog messages using this pattern as well?

Reply:
Sure, it is needed for consistency. I'll push changes in a while.